typed-racket-more/typed/racket/private/gui-types.rkt

```diff
@@ -1592,6 +1592,8 @@
                [stretchable-height Any #:optional])
          [append (String -> Void)]
          [delete (Integer -> Void)]
+         [on-reorder (Listof Exact-Nonnegative-Integer)]
```

Comment:
according to https://docs.racket-lang.org/gui/tab-panel_.html?q=on-reorder%20#%28meth._%28%28%28lib._mred%2Fmain..rkt%29._tab-panel~25%29._on-reorder%29%29, *on-reorder* is a *method* that takes a list of exact-nonnegative-integer? and returns void. So is *on-close-request*

That's why the test failed.

Reply:
Ah! Right, I can't believe I didn't notice that! Glad to have another pair of eyes on this.